src/utilities/timeline.ts

```diff
@@ -196,6 +196,38 @@ export function createVerticalGuide(doyTimestamp: string, verticalGuides: Vertic
   };
 }
 
+/*
+  Returns a new horizontal guide
```

Comment:
Make sure the format of these doc comments are correct (we should do a pass over these in this file to make sure they are correct):

```js
/**
 * Returns a new horizontal guide
 */
```

Reply:
Good catch